sdk/app/azure-mgmt-app/azure/mgmt/app/models/_container_apps_api_client_enums.py

```diff
@@ -7,18 +7,17 @@
 # --------------------------------------------------------------------------
 
 from enum import Enum
-from six import with_metaclass
 from azure.core import CaseInsensitiveEnumMeta
 
 
-class AccessMode(with_metaclass(CaseInsensitiveEnumMeta, str, Enum)):
+class AccessMode(str, Enum, metaclass=CaseInsensitiveEnumMeta):
```

Comment:
nit: this package is deprecated so we don't need care about it even if CI fails for it.